slick/src/main/scala/slick/jdbc/SQLiteProfile.scala

```diff
@@ -98,14 +98,26 @@ trait SQLiteProfile extends JdbcProfile {
     override def createColumnBuilder(tableBuilder: TableBuilder, meta: MColumn): ColumnBuilder = new ColumnBuilder(tableBuilder, meta) {
       /** Regex matcher to extract name and length out of a db type name with length ascription */
       final val TypePattern = "^([A-Z]+)(\\(([0-9]+)\\))?$".r
+      final val TimestampPattern = """^(?:"(.*)")|([0-9]+)$""".r
       private val (_dbType,_size) = meta.typeName match {
         case TypePattern(d,_,s) => (d, Option(s).map(_.toInt))
       }
       override def dbType = Some(_dbType)
       override def length = _size
       override def varying = dbType == Some("VARCHAR")
-      override def default = meta.columnDef.map((_,tpe)).collect{
+      override def default = meta.columnDef.map((_,tpe)).collect{ 
         case ("null",_)  => Some(None) // 3.7.15-M1
+        case (v , "java.sql.Timestamp") => v match{
+          case TimestampPattern(ts , ts_unix) => Some(Some( 
+            if( ts_unix != null )
+              new java.sql.Timestamp(ts_unix.toLong)
+            else
+              try java.sql.Timestamp.valueOf(ts.replace("\"","")) catch{
+                case e: java.lang.IllegalArgumentException => 
+                    new java.sql.Timestamp(javax.xml.bind.DatatypeConverter.parseDateTime(ts).getTime.getTime) 
```

Comment:
It seems that `javax.xml.bind.DatatypeConverter.parseDateTime` is not able to parse `2013-10-07 04:23:19.120-04:00`.